### PR TITLE
Add JetBrains Toolbox extension

### PR DESCRIPTION
**Pull request by gdsmith**
## Description

Search and open JetBrains Toolbox projects

## Type of change

- [x] New extension

## Screenshot

screenshot

## Checklist

- [x] I read the [contribution guidelines](contributing.md)

### CODE REVIEW

File: extensions/jetbrains/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "name": "jetbrains",
+  "title": "JetBrains",
+  "description": "Search and open JetBrains Toolbox projects",
+  "icon": "jb.png",
+  "author": "gdsmith",
+  "commands": [
+    {
+      "name": "recent",
+      "title": "JetBrains Recent Projects",
+      "subtitle": "",
```
Comment (PitNikola) [inline extensions/jetbrains/package.json:11]: Suggestion:
Change title to "Search Recent Projects"
Subtitle to "JetBrains"

**Comment by gdsmith** [comment]
@PitNikola thanks for the review.

> I installed AppCode to test it (I didn't have any JetBrains product), and experienced two issues:

Did you install it via JetBrains Toolbox or as a standalone app?

**Comment by gdsmith** [comment]
@PitNikola Okay, updated as this was currently only designed to work with Jetbrains Toolbox installations

**Comment by pyinto** [comment]
@gdsmith I have toolbox and pycharm (via toolbox) installed. The plugin still shows me this error message:
> No JetBrains applications found. Please check that you have JetBrains Toolbox and at least one IDE installed.

**Comment by gdsmith** [comment]
In Toolbox / Settings there's a _Generate shell scripts_ option, do you have that enabled?

**Comment by pyinto** [comment]
@gdsmith it is enabled, I enabled it myself, first thing I've tried after I found that extension does not work

**Comment by SebastiaanKloos** [comment]
@pyinto I had the same issue. Change the path to `~/.config/jetbrains/bin`. That should work!

**Comment by gdsmith** [comment]
Working on a fix. Had not realised I had customised that path.

**Comment by gdsmith** [comment]
@pyinto @SebastiaanKloos updated extension version is released if you can try that and see if it solves your issues

**Comment by SebastiaanKloos** [comment]
> @pyinto @SebastiaanKloos updated extension version is released if you can try that and see if it solves your issues

Yes it works great! The config option is also very nice! Many thanks 🎉

**Pull request merged**